### ISSUE TITLE
All autocollected dependency requests (fetch, xhr) are labeled as "Ajax"

### ISSUE THREAD

**Issue by markwolff** [Microsoft/ApplicationInsights-JS#758]
![image](https://user-images.githubusercontent.com/5515583/51413649-765ba180-1b24-11e9-953b-42f604e9b1d9.png)

https://github.com/Microsoft/ApplicationInsights-JS/blob/4c92246fafd603d498385355ff05776b78d549fa/AppInsightsCommon/src/Telemetry/RemoteDependencyData.ts#L54

All auto tracked requests are listed as `Ajax` in portal. Now that we support collection of `fetch` requests, we should label it properly. There are 2 ways I can think of to do this without causing much disruption on other partner dependent packages.

1. Use existing `type` field in `IDependencyTelemetry`. Care must be taken that this doesn't break partner dependencies, like 1p plugin.
2. Add a `RequestType` field to `properties` like how it was done here and read it in channel. https://github.com/Microsoft/ApplicationInsights-JS/blob/4c92246fafd603d498385355ff05776b78d549fa/extensions/applicationinsights-dependencies-js/src/ajax.ts#L497

Option 2 seems less disruptive but if `type` can be used instead, that would be probably preferred.

**Comment by jpiyali** [comment]
@hectorhdzg  - can you provide input on what customers usually fill in for 1P for type?

**Comment by hectorhdzg** [comment]
We use to auto populate this field with "Webservice", is just some string categorization, according to JSLL docs this could contain words like SqlServer, WebService, etc. In 1st party world autocollection of QoS events were not easy so must people implemented this kind of events calling the APIs directly and specifying whatever value they wanted as type

**Comment by jpiyali** [comment]
was there no auto capture in 1p SDK?

**Comment by hectorhdzg** [comment]
In JSLL there was not auto collection of QoS events

**Comment by jpiyali** [comment]
then, is it okay if we populate fetch/ajax for type in auto collection scenarios?

**Comment by hectorhdzg** [comment]
That is ok, just ensure people can override the type when calling the APIs directly

**Comment by markwolff** [comment]
I am going to use the existing `type` field and move existing `getPathName` logic to `name`. 
```ts
name: ajaxData.getPathName(),
type: "Fetch"
```
This seems to match old the `ajax.ts`, where the 3rd arg (`getPathName`) is read as `commandName` into the `data` part B field
https://github.com/Microsoft/ApplicationInsights-JS/blob/4c92246fafd603d498385355ff05776b78d549fa/JavaScript/JavaScriptSDK/ajax/ajax.ts#L244-L251
